modules/tcmalloc/0.0.0-20250331-43fcf6e/MODULE.bazel

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 The TCMalloc Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module(
+    name = "tcmalloc",
+    version = "0.0.0-20250331-43fcf6e",
+    compatibility_level = 0,
+    repo_name = "com_google_tcmalloc",
+)
+
+bazel_dep(name = "abseil-cpp", version = "20240722.0", repo_name = "com_google_absl")
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "fuzztest", version = "20241028.0", repo_name = "com_google_fuzztest")
+bazel_dep(name = "google_benchmark", version = "1.8.3", repo_name = "com_github_google_benchmark")
```

Comment:
Does this have to be a required dependency? And if so can it be updated to `1.9.2` to avoid the `rules_foreign_cc` dependency?

Reply:
Ah, disregard, I see there's [upstream support for bzlmod](https://github.com/google/tcmalloc/blob/6fa585bee4bd6a0aefed5ec0142e38fce9ea8b8e/MODULE.bazel)

Reply:
yea we should fix upstream. it looks like it has to be required since it's mixed in with the other BUILD rules, i imagine an update would be fine

Reply:
I filed https://github.com/google/tcmalloc/pull/277 but there are other oddities that should probably be updated as well.